tests/CMakeLists.txt

```diff
@@ -189,6 +189,7 @@ if(WITH_PYTHON)
   add_fdb_test(TEST_FILES rare/ConflictRangeRYOWCheck.toml)
   add_fdb_test(TEST_FILES rare/CycleRollbackClogged.toml)
   add_fdb_test(TEST_FILES rare/CycleWithKills.toml)
+  add_fdb_test(TEST_FILES rare/DDUnitTests.toml UNIT)
```

Comment:
These unit tests will already be run regularly in the `AllUnitTests.toml` and `RandomUnitTest.toml` tests, so IMO this test file should be removed from the PR.

Reply:
Are there situations where we want to run tests for a particular area alone?

Reply:
Not in nightly simulation testing, but if you want to run tests on a specific area while debugging, the `SpecificUnitTest.toml` file can be modified and used.